templates/legal/equal-employment-opportunity-commitment.html

```diff
@@ -0,0 +1,28 @@
+{% extends "legal/_base_legal.html" %}
+
+{% block title %}Equal Employment Opportunity Commitment{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1_ExS3fRc8odw8extzVeZWkrZU78DekJ3Ei_dUOayGLI/edit{% endblock meta_copydoc %}
+
+{% block content %}
+<div class="p-strip is-deep">
+  <div class="row">
+    <div class="col-8">
+      <h1>Canonical Equal Employment Opportunity Commitment</h1>
+
+      <p class="p-text--small">In compliance with USA Code: 41 C.F.R. §§ 60-300.44(a); 60-741.44(a)</p>
+
+      <p>It is the policy of Canonical USA Inc. (the  “Company”) to provide equal employment opportunity to all employees and applicants without regard to race, religion, sex, gender identity, sexual orientation, national origin, physical or mental disability, status as a protected veteran, or any other characteristics protected by law.  This policy governs all terms and conditions of employment, including recruiting, hiring, training, promotion, demotion, transfer, layoff, compensation, discipline, and termination.</p>
```

Comment:
```suggestion
      <p>It is the policy of Canonical USA Inc. (the  "Company") to provide equal employment opportunity to all employees and applicants without regard to race, religion, sex, gender identity, sexual orientation, national origin, physical or mental disability, status as a protected veteran, or any other characteristics protected by law.  This policy governs all terms and conditions of employment, including recruiting, hiring, training, promotion, demotion, transfer, layoff, compensation, discipline, and termination.</p>
```
Straighten the speech marks